apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -2,22 +2,50 @@
 import { useIntl } from "react-intl";
 import { useQuery } from "urql";
 
-import { Pending } from "@gc-digital-talent/ui";
+import { Pending, ResourceBlock } from "@gc-digital-talent/ui";
 import { ROLE_NAME } from "@gc-digital-talent/auth";
-import { User, graphql } from "@gc-digital-talent/graphql";
+import {
+  graphql,
+  ApplicantDashboard_QueryQuery as ApplicantDashboardQueryType,
+} from "@gc-digital-talent/graphql";
 import { commonMessages } from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
 
+import useRoutes from "~/hooks/useRoutes";
 import SEO from "~/components/SEO/SEO";
 import { getFullNameHtml } from "~/utils/nameUtils";
 import RequireAuth from "~/components/RequireAuth/RequireAuth";
 import Hero from "~/components/Hero";
+import { isVerifiedGovEmployee } from "~/utils/userUtils";
+
+import CareerDevelopmentTaskCard from "./components/CareerDevelopmentTaskCard";
 
 export interface DashboardPageProps {
-  currentUser?: User | null;
+  currentUser?: ApplicantDashboardQueryType["me"];
```

Comment:
I think we should be using a fragment here so we can be more confident in the types

Reply:
Yeah, I could be using fragments better here and throughout
Big commit in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/dff254d46efbfb5f6a266186d3eff757e12f4239